# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/LocalAccessible.cpp
# Commit: 48dc94da9565
# Full Hash: 48dc94da9565d7f804092209a27821d729ccde60
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-04-02 09:44:13
# Regressor Bug: 1735970
# File Overlap Count: 1
# Description:
#   Bug 1735970 part 3: Add TableAccessibleBase and TableCellAccessibleBase. r=morgan
#   
#   These are needed to support RemoteAccessible tables.
#   Stuff specific to LocalAccessible is still in TableAccessible and TableCellAccessible, which now inherit from the new Base classes.
#   
# ==============================================================================

diff -r 477c59860bba -r 48dc94da9565 accessible/generic/LocalAccessible.cpp
--- a/accessible/generic/LocalAccessible.cpp	Fri Apr 01 09:49:56 2022 +0000
+++ b/accessible/generic/LocalAccessible.cpp	Fri Apr 01 09:49:57 2022 +0000
@@ -3552,3 +3552,9 @@
 
   aValue.Append(mKey);
 }
+
+TableAccessibleBase* LocalAccessible::AsTableBase() { return AsTable(); }
+
+TableCellAccessibleBase* LocalAccessible::AsTableCellBase() {
+  return AsTableCell();
+}